zrml/global-disputes/src/lib.rs

```diff
@@ -189,6 +191,8 @@ mod pallet {
         OutcomeAlreadyExists,
         /// The outcome specified is not present in the voting outcomes.
         OutcomeDoesNotExist,
+        /// Submitted outcome does not match market type.
+        OutcomeMismatch,
```

Comment:
Not able to find test covering this error. Could you please add it or explain why not needed?

Reply:
fed893f0125d44fb28e8d25d4e55b5b05aeb875a